featuretools/primitives/standard/transform/datetime/season.py

```diff
@@ -0,0 +1,55 @@
+from datetime import date
+
+import pandas as pd
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Categorical, Datetime
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class Season(TransformPrimitive):
+    """Determines the season of a given datetime.
+        Returns winter, spring, summer, or fall.
```

Comment:
Is there a reason for dropping the `hemisphere` arg from this primitive? If we don't include it, we should probably update the docstring to indicate this only works for locations in the northern hemisphere.

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/2460/commits/7b162d66842596a2eb47407388ad6c896c7253ce